CMakeLists.txt

```diff
@@ -79,29 +76,26 @@ target_link_libraries( ACEtk
 # ACEtk : python bindings
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-include_directories( python/src/ )
-
-pybind11_add_module( ACEtk.python
-    EXCLUDE_FROM_ALL
+if(ACEtk.python)
+  pybind11_add_module( ACEtk.python 
     python/src/ACEtk.python.cpp
     python/src/Data.python.cpp
     python/src/Table.python.cpp
-)
-target_link_libraries( ACEtk.python PRIVATE ACEtk )
-target_compile_options( ACEtk.python PRIVATE "-fvisibility=hidden" )
-set_target_properties( ACEtk.python PROPERTIES OUTPUT_NAME ACEtk )
-set_target_properties( ACEtk.python PROPERTIES COMPILE_DEFINITIONS "PYBIND11" )
-set_target_properties( ACEtk.python PROPERTIES POSITION_INDEPENDENT_CODE ON)
+  )
+  target_link_libraries( ACEtk.python PRIVATE ACEtk )
+  target_include_directories( ACEtk.python PRIVATE python/src )
+  target_compile_options( ACEtk.python PRIVATE "-fvisibility=hidden" )
+  set_target_properties( ACEtk.python PROPERTIES OUTPUT_NAME ACEtk )
+  set_target_properties( ACEtk.python PROPERTIES COMPILE_DEFINITIONS "PYBIND11" )
+  set_target_properties( ACEtk.python PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endif()
 
 #######################################################################
 # Top-level Only
 #######################################################################
 
 if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
```

Comment:
Looking at this again, it doesn't make sense to put this in a top-level only block if it's controlled by a CMake variable.  But that's not related to your changes, so I'll just add this as a todo for myself!